indico/modules/events/abstracts/templates/abstract.html

```diff
@@ -1,9 +1,21 @@
-{% extends 'events/abstracts/management/_base.html' %}
+{% extends 'events/abstracts/management/_base.html' if management else 'events/abstracts/display/_base.html' %}
 
 {% from 'events/abstracts/abstract/public.html' import render_abstract_public %}
 {% from 'events/abstracts/abstract/judge.html' import render_decision_box, render_reviewed_for_tracks_box %}
 {% from 'events/abstracts/abstract/timeline.html' import render_timeline %}
 
+{% block back_button_url -%}
+    {%  if management %}
```

Comment:
too much space. also, it's important that you do strip surrounding whitespace here so you don't get a back button with a url containing whitespace